test/browser/nested/element.vdiff.js

```diff
@@ -0,0 +1 @@
+import '../element.vdiff.js';
```

Comment:
I've found having this second nested test file was really helpful for all the file manipulation testing, so figured I'd just add it. Could also help with the deleting/creating/rolling up reports testing!